typescript/cli/examples/submit/strategy/gnosis-tx-builder-ica-strategy.yaml

```diff
@@ -0,0 +1,14 @@
+polygonamoy:
+  submitter:
+    type: interchainAccount
+    # The origin chain for the ICA transactions
+    chain: basesepolia
+    # The destination chain for the transactions
+    destinationChain: polygonamoy
+    # The owner of the remote ICA account
+    owner: '0x32e02dDB233958834fcB5E93b11E254F8D1EAd1b'
+    internalSubmitter:
```

Comment:
first time reading this and with admittedly weak ICA experience, this submitter + internalSubmitter relationship is confusing to me. 

i would add a comment here on it's intention